### PR TITLE
build: correct ordering of `include`

### PR DESCRIPTION
**Pull request by compnerd**
Correct the ordering that @bnbarham pointed out post-commit.  Ensure that we adjust the `CMAKE_MODULE_PATH` prior to the use of `include` which will reference local modules.

### CODE REVIEW

**Comment by compnerd** [comment]
@rauhul - turns out that I do not have commit access on the repository, I would appreciate it if you could assist with that as well 😀

CC: @bnbarham @xedin

**Comment by compnerd** [comment]
@swift-ci please test

**Review by bnbarham — APPROVED** [review]

**Comment by natecook1000** [comment]
LGTM! @compnerd will this require a new release?

**Comment by compnerd** [comment]
@natecook1000 - I don't think that the previous change went into 1.3.0, so I think we should be okay.  If it did make it into 1.3.0, we should do a 1.3.1.

**Comment by compnerd** [comment]
@natecook1000, @rauhul - anything else to do before we can merge this?

**Comment by natecook1000** [comment]
Nope! Thanks @compnerd!

**Pull request merged**
